trick_source/trick_utils/compareFloatingPoint/test/.gitignore

```diff
@@ -0,0 +1,2 @@
+*.o
+Interpolator_unittest
```

Comment:
I'm thinking this should be dbl_is_near_unittest and flt_is_near_unittest? Or *_unittest

Reply:
Fixed.